apps/challenges/challenge_config_utils.py

```diff
@@ -378,6 +378,13 @@ def validate_challenge_title(self):
         if not challenge_title or len(challenge_title) == 0:
             message = self.error_messages_dict.get("missing_challenge_title")
             self.error_messages.append(message)
+        else:
+            # THese are allowed characters
+            allowed_pattern = r"^[a-zA-Z0-9\s\-_\.\,\'\!\&]+$"
+            match = re.match(allowed_pattern, challenge_title)
+            if not match:
+                message = "Challenge title can only contain alphanumeric characters, spaces, hyphen, underscore, period, comma, apostrophe, exclamation point, and ampersand."
```

Comment:
@AishwaryJain07 Instead of putting the message here, i think we should keep it with all the other error messages in `error_message_dict`

Reply:
@Suryansh5545 done